dev-python/safety/safety-3.5.2.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{11..13} )
+
+DISTUTILS_USE_PEP517="hatchling"
+
+inherit distutils-r1
+
+DESCRIPTION="Scan dependencies for known vulnerabilities and licenses."
+HOMEPAGE="https://safetycli.com"
+SRC_URI="https://github.com/pyupio/safety/archive/refs/tags/${PV}.tar.gz -> ${P}.gh.tar.gz"
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+
+PATCHES=(
```

Comment:
Include upstream PR/commit links to patches.

Reply:
The patches are variations of upstream patches. Not direct copy and paste.

Reply:
The idea here is that if they were direct upstream commits we might not need a comment. e.g.

```
From abbf43c55beca86b963d169eb746b59f11dda1ae Mon Sep 17 00:00:00 2001
From: Oz Tiram <oz.tiram@gmail.com>
Date: Sat, 5 Jul 2025 13:43:13 +0200
Subject: [PATCH] fixes for click 8.2
```

This has a default comment claiming to be commit abbf43c55beca86b963d169eb746b59f11dda1ae but no such commit actually exists.

Please include a comment saying which upstream PR it was a variation on. That is the whole point of the review request.

Reply:
OK. I understand the issue. The commits in the patches were variations of upstream commit, on a local branch I had.
In the meanwhile, version 3.6.0 was release, so some of the patches are not needed. The patches for nltk, do have an open PR upstream. I added a comment in the patch pointing to it.
As for the dropped patches, well, I also updated safety to version 3.6.0.